examples/eks/aws-cloudwatch/otel-sidecar.yaml

```
@@ -26,7 +26,7 @@ spec:
     spec:
       containers:
         - name: aws-otel-emitter
-          image: "aottestbed/aws-otel-collector-sample-app:java-0.1.0"
+          image: "aottestbed/aws-otel-collector-java-sample-app:0.9.0"
```

```suggestion
          image: "public.ecr.aws/aws-otel-test/aws-otel-java-test-spark:v0.11.0"
```
Please use the most recent image version of the spark sample app.